lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -119,6 +119,25 @@ static StringRef getNameForObjC(const ValueDecl *VD,
   return VD->getName().str();
 }
 
+static Optional<StringRef>
+printSwiftEnumElemNameInObjC(const EnumElementDecl *EL, llvm::raw_ostream &OS,
```

Comment:
If taking Option 1, the caller of `printSwiftEnumElemNameInObjC ` will need to call `getNameForObjC` again to get the specified name.  Option 2 defies the purpose of declaring `printSwiftEnumElemNameInObjC ` because we want it to encapsulate the logic of collecting names.